base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,23 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(function, ArgumentTypes::Type, arguments...)
```

Comment:
put back `f` since `function` is a keyword?
We usually don't capitalize argument names, perhaps change to argtypes? Also, `::Type` should be `::Tuple`?